test/distributed/org/apache/cassandra/distributed/test/InternodeEncryptionEnforcementTest.java

```diff
@@ -161,10 +147,11 @@ public void testOutboundConnectionsAreInterruptedWhenAuthFails() throws IOExcept
 
                 // There should be no outbound connections as authentication fails.
                 OutboundConnections outbound = getOnlyElement(MessagingService.instance().channelManagers.values());
-                assertTrue(!outbound.small.isConnected() && !outbound.large.isConnected() && !outbound.urgent.isConnected());
+                Awaitility.await().until(() -> !outbound.small.isConnected() && !outbound.large.isConnected() && !outbound.urgent.isConnected());
```

Comment:
Looks like the default wait is 10s - should probably be enough time.

Reply:
Out of 500 runs none failed, so It should be okay.